.github/workflows/one-click-hosting-release.yml

```diff
@@ -0,0 +1,140 @@
+name: One Click Hosting Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      pre_release:
+        type: boolean
+        description: 'Pre-release?'
+        default: true
+        required: false
+    branches:
+      - main
+env:
+  CHANNEL: 'cloud'
+  CHANGELOG_FILE: 'temp-changelog.txt'
+
+jobs:
+  release:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: checkout branch
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.ref }}
+      - name: Get Configuration
+        id: configuration
+        uses: rgarcia-phi/json-to-variables@main
+        with:
+          filename: './.github/workflows/config.json'
+          prefix: config
+      - name: Test permissions
+        uses: ./.github/workflows/permissions
+        with:
+          ENVIRONMENT: ${{ env.config_deployment_environment }}
+          DEPLOYMENT_PERMITTED: ${{ env.config_deployment_permitted }}
+          DEPLOYMENT_REPOSITORY_OWNER: ${{ env.config_deployment_repository_owner }}
+      - name: Install Node.js 18.x
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18.x
+      - name: Is pre-release?
+        id: is_pre_release
+        run: |
+          if [ ${{ github.event.inputs.pre_release }} == true ]; then
+            POSTFIX="-test"
+            echo "This is a prerelease execution"
+          else
+            POSTFIX=""
+            echo "This is a release execution"
+          fi
+          echo "POSTFIX=${POSTFIX}" >> $GITHUB_ENV
```

Comment:
Don't we want to do this in `env-vars-configure`?

Reply:
Also, we pass this variable around a lot, when it actually means "when you generate a package version, add this string". Can't we just generate the package version names and use them without other code areas aware of this?

Reply:
1. This step was moved to `env-vars-configure` file.
2. What I want to achieve is a package version name that contains the `POSTFIX` and the  `NEW_CHANNEL_VERSION` E.g, `3.16.0-cloud5-test` so that releases won't be dependent on pre-release tags.
at the moment I don't see other another way to overwrite the package version name. 

I'd love to have a meeting and discuss that.